corehq/apps/hqwebapp/management/commands/resource_static.py

```diff
@@ -4,35 +4,20 @@
 
 from django.conf import settings
 from django.contrib.staticfiles import finders
-from django.core import cache
 from django.core.management.base import BaseCommand
 
 import yaml
 
 from dimagi.utils import gitinfo
```

Comment:
I think that `gitinfo` is no longer used here

Reply:
8db3a6e489908e85a76a8353435d5058f1440ae0